Documentation/ceph-examples.md

```diff
@@ -13,14 +13,12 @@ indent: true
 
 # Ceph Examples
 
-Configuration for Rook and Ceph comes in many shapes and sizes. While we have done everything possible to make configuring storage
-simple, you will need to decide what settings work for your environment. The settings on the operator and the
-Rook Custom Resource Definitions (CRDs) are flexible. To get started, we have created several common configurations.
+Configuration for Rook and Ceph can be configured in multiple ways to provide block devices, shared file system volumes or object storage in a kubernetes namespace.We have provided several examples to simplify storage setup, but remember there are many tunables and you will need to decide what settings work for your use case and environment. 
 
-See the **[Ceph examples][example yaml files](https://github.com/rook/rook/blob/{{ branchName }}/cluster/examples/kubernetes/ceph)** folder for the yaml files.
+See the **[Ceph examples][example yaml files](https://github.com/rook/rook/blob/{{ branchName }}/cluster/examples/kubernetes/ceph)** folder for all the rook/ceph setup example spec files.
```

Comment:
Can you remove the `[Ceph examples]` text? It should not have been included originally and can be seen on the [master doc page](https://rook.io/docs/rook/master/ceph-examples.html) as not making sense.